metron-deployment/vagrant/full-dev-platform/run.sh

```diff
@@ -17,4 +17,4 @@
 # limitations under the License.
 #
 
-vagrant --ansible-skip-tags="solr" up
+vagrant --ansible-skip-tags="solr,yaf" up
```

Comment:
That was my intention. No point in waiting for download and compilation if we're not using it. Can always override/rerun after.